cabal-install/Distribution/Solver/Types/ComponentDeps.hs

```diff
@@ -22,6 +22,7 @@ module Distribution.Solver.Types.ComponentDeps (
   , fromList
   , singleton
   , insert
+  , zip
```

Comment:
...this causes a clash in the testsuite which appears to import ComponentDeps unqualified:

```
tests/UnitTests/Distribution/Client/InstallPlan.hs:116:39:
    Ambiguous occurrence ‘zip’
    It could refer to either ‘Data.List.zip’,
                             imported from ‘Data.List’ at tests/UnitTests/Distribution/Client/InstallPlan.hs:20:1-16
                             (and originally defined in ‘GHC.List’)
                          or ‘CD.zip’,
                             imported from ‘Distribution.Solver.Types.ComponentDeps’ at tests/UnitTests/Distribution/Client/InstallPlan.hs:14:1-62
                             (and originally defined
                                at Distribution/Solver/Types/ComponentDeps.hs:127:1-3)
```